isisd/isisd.c

```diff
@@ -3301,7 +3361,7 @@ void isis_restart_write_overload_time(struct isis_area *isis_area,
 	}
 
 	json_object_int_add(json_area, "overload_time",
-			    isis_area->overload_on_startup_time);
+			    overload_time);
 	json_object_to_file_ext(filepath, json, JSON_C_TO_STRING_PRETTY);
```

Comment:
Is it a fix for ttps://github.com/FRRouting/frr/pull/11980 ?

Please move that to a separate commit.

Reply:
Removed change